client/src/pages/reports/Minimal.js

```diff
@@ -394,7 +395,14 @@ const ReportMinimal = props => {
               </Fieldset>
 
               <Fieldset title="Meeting attendees">
-                <AttendeesTable attendees={report.attendees} disabled />
+                <AttendeesTable
+                  attendees={report.attendees}
+                  disabled
+                  showSensitive={
+                    report.reportSensitiveInformation &&
+                    report.reportSensitiveInformation.text
```

Comment:
This same condition now appears twice in this render, better extract it into an aptly named local variable.

Reply:
I am thinking about using conditional chaining instead (thus replacing it by report?reportSensitiveInformation?text)

Reply:
It's still the same repeated expression in more than one place (where it has the same meaning), so still better to extract it into a single variable IMHO.

Reply:
Alright